packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -109,12 +109,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
 
   @autobind
   public dismissSuggestions() {
-    // Select the first suggestion if one is available when user leaves.
-    let selectItemFunction = () => {
-      if (this.suggestionStore.hasSelectedSuggestion() && this.state.suggestedDisplayValue) {
-        this.addItemByIndex(0);
-      }
-    };
+    let selectItemFunction = this.props.onDismiss ?
```

Comment:
Typically our callback functions have zero effect on the functionality of the component. If you want a way to suppress default onDismiss functionality, I suggest using ev.defaultPrevented approach like you see here https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx#L491


This allows people to use the callback to trigger some other function without affecting the component, or prevent default behavior if needed.

Reply:
Another suggestion would be to pass in the currently selected item into the onDismiss so that you can decide what to do based on the value passed in
` this.suggestionStore.currentSuggestion.item`